### PR TITLE
[Merged by Bors] - api: v2alpha1 network service

### PR DESCRIPTION
**Pull request by kacpersaw**
getting network info:
```
> grpcurl -plaintext -d '{}' 127.0.0.1:9092 spacemesh.v2alpha1.NetworkService.Info

{
  "genesisTime": "2023-09-27T23:00:00Z",
  "layerDuration": "300s",
  "genesisId": "e/LCAhFkqOZG+CWN3TGAp8pKcnA=",
  "hrp": "stest",
  "effectiveGenesisLayer": 575,
  "layersPerEpoch": 288
}
```

### CODE REVIEW

File: api/grpcserver/v2alpha1/interface.go [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+import (
+	"time"
+
+	"github.com/spacemeshos/go-spacemesh/common/types"
+)
+
+// genesisTimeAPI is an API to get genesis time and current layer of the system.
+type genesisTimeAPI interface {
+	GenesisTime() time.Time
```
Comment (poszu) [inline api/grpcserver/v2alpha1/interface.go]: Does it make sense to have a method to get the genesis time? It's constant and could be passed to `NewNetworkService()` like other constants.

**Review by poszu — DISMISSED** [review]

**Review by poszu — APPROVED** [review]

**Comment by kacpersaw** [comment]
bors try

**Comment by kacpersaw** [comment]
bors merge

**Comment by kacpersaw** [comment]
bors merge

**Comment by kacpersaw** [comment]
bors merge

**Comment by kacpersaw** [comment]
bors merge

**Pull request closed without merging**